overview/glossary.md

```diff
@@ -0,0 +1,33 @@
+| Term                                                                                  | Definition                                                                                                                                                                                                |
+| ---                                                                                   | ---                                                                                                                                                                                                       |
+| Attestation                                                                           | A verification of something as true or authentic by evidence.  In uPort attestations are cryptographically signed claims.                                                                                 |
+| Application Identity                                                                  | Just like a user has a uPort identity, applications and dApps that interact with uPort users need their own identity a.k.a (key/pair)                                                                     |
+| Claim                                                                                 |  A claim is a record of an attestation that has been verified.                                                                                                                                            |
+| Credentials                                                                           | Credentials are pieces of information about a uPort user that may be requested and/or verified by applications.  They are, in their essence, signed claims that are stored with the uPort user's profile. |
+| dAPP                                                                                  | Distributed Application, generally one that relies on serverless architectures.                                                                                                                           |
+| DID                                                                                   | Decentralized Identity Document as defined by the [W3C Community Group](https://w3c-ccg.github.io/did-spec/)                                                                                              |
```

Comment:
DID stand for Decentralized Identifier and DID doc is a separate term as far as I know